src/app/components/Figure/index.jsx

```diff
@@ -28,23 +14,21 @@ const Image = styled.img`
   width: 100%;
 `;
 
-const Figure = ({ src, alt, copyrightHolder, caption }) => (
+const Figure = ({ src, alt, children }) => (
   <StyledFigure>
     <Image src={src} alt={alt} />
-    {renderCopyright(copyrightHolder)}
-    {renderCaption(caption)}
+    {children}
   </StyledFigure>
 );
 
 Figure.propTypes = {
   alt: string.isRequired,
   src: string.isRequired,
-  copyrightHolder: string.isRequired,
-  caption: string,
+  children: shape({ element }),
```

Comment:
Would appreciate feedback on whether this is logical or is this should be a type of `node`? I'm not sure as this can conditionally be either nothing, Caption, Copyright or Caption & Copyright.

Reply:
This is logical. The only big problem I have with this is that it introduces another layer of abstraction. Technically we could even pass another container as a child which then renders another component. Thats a lot of steps for rendering a Figure. Ideally this logic would be handled by the Figure container and not the component but in this case it has to be done by the component since it is a wrapper.

Reply:
This implies that children has the shape of an object with elements. The children you are passing to this component do not match the `shape({ element})`. One way to check this is the case is to run `npm run dev`, visit a page that has an image, e.g. http://localhost:7080/news/articles/c0000000027o and see the proptype error in the console. 
```
Warning: Failed prop type: Invalid prop `children` of type `array` supplied to `Figure`, expected `object`.
```